go.mod

```diff
@@ -4,3 +4,5 @@ require (
 	github.com/onsi/ginkgo v1.8.0
 	github.com/onsi/gomega v1.5.0
 )
+
+go 1.13
```

Comment:
We don't really need that info as nothing changed here

Reply:
This is what go mod does automatically - not going to fight it on this matter